### PR TITLE
Change /join and /connect links to MeshDB join form

### PR DESCRIPTION
**Pull request by LydonThorpe**
Part of MeshDB deployment. Please approve and wait for roll-out to merge.
Changes /connect and /join to point to the MeshDB join form. Changes the "Get Connected" button in the header.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -11,3 +11,27 @@ HUGO_VERSION = "0.69.0"
...
+from = "/zh/join"
+to = "https://forms.nycmesh.net/join"
+status = 302
+force = true
+
+[[redirects]]
+from = "/connect"
+to = "https://forms.nycmesh.net/join"
+status = 302
+force = true
```
Comment (Andrew-Dickinson) [inline netlify.toml:37]: Can we do these in `static/_redirects` instead?

**Review by Andrew-Dickinson — CHANGES_REQUESTED** [review]

**Comment by LydonThorpe** [comment]
moved redirects to /static/_redirects based on feedback

File: static/_redirects [hunk 2]
```diff
@@ -11,3 +11,7 @@
 /panorama/1.7miles.png /img/1.7miles.png
 /network-status https://status.mesh.nycmesh.net
 /status https://status.mesh.nycmesh.net
+/join https://forms.nycmesh.net/join
+/es/join https://forms.nycmesh.net/join
```
Comment (Andrew-Dickinson) [inline static/_redirects:15]:
```suggestion
/es/join https://forms.nycmesh.net/es/join/
```

File: static/_redirects [hunk 3]
```diff
@@ -11,3 +11,7 @@
...
+/zh/join https://forms.nycmesh.net/join
```
Comment (Andrew-Dickinson) [inline static/_redirects:16]:
```suggestion
/zh/join https://forms.nycmesh.net/zh/join/
```

**Review by Andrew-Dickinson — APPROVED** [review]

**Pull request merged**